src/configurations/destinations/fb_custom_audience/ui-config.json

```diff
@@ -464,7 +464,22 @@
                     ],
                     "default": "NA"
                   }
-                ]
+                ],
+                "preRequisites": {
```

Comment:
is this added for only field or all fields which needs to move to connection config

Reply:
It is added for the entire group which contains all the fields moved to connection settings